NEWS

```diff
@@ -66,17 +66,18 @@ CHANGES WITH 256 in spe:
 
         * A new 'systemd-vpick' binary has been added. It implements the new
           vpick protocol, where a .v directory may contain multiple files with
-          a version embedded in the name, and they may be ordered by version
-          and the newest one can be reliably selected.
+          a version following the UAPI version format specification the
+          embedded in the name. The files are ordered by version and the newest
```

Comment:
Can't interpret this?

Reply:
Removed the superfluous "the" before embedded and added commas for added clarity.